Using system expat

Patch written by Kacper Kowalik <xarthisius.kk@gmail.com>

--- setup.py
+++ setup.py
@@ -13,14 +13,9 @@
 
 iterparse_ext = Extension(
     "vo.iterparser",
-    ["src/iterparse.c",
-     "src/expat/xmlparse.c",
-     "src/expat/xmlrole.c",
-     "src/expat/xmltok.c",
-     "src/expat/xmltok_impl.c",
-     "src/expat/xmltok_ns.c"],
+    ["src/iterparse.c",],
     define_macros=defines,
-    include_dirs=["src", "src/expat"])
+    libraries=["expat"])
 
 setup(name="vo",
       version="0.5",
